tests/functional/pv/pvc_snapshot/test_snapshot_restore_with_different_access_mode.py

```diff
@@ -22,6 +23,7 @@
 @skipif_ocs_version("<4.6")
 @skipif_ocp_version("<4.6")
 @polarion_id("OCS-2410")
+@bugzilla("2249601")
```

Comment:
The main issue in the BZ is the presence of `pool: <>` parameter in cephfs SC and use of ROX clone.
Are we using this pool param in the cephfs SC? and I hope the parent pvc is of type RWX

Reply:
It is present in cephfs SC.
yaml from external storagecluster.

allowVolumeExpansion: true
apiVersion: [storage.k8s.io/v1](http://storage.k8s.io/v1)
kind: StorageClass
metadata:
  annotations:
    description: Provides RWO and RWX Filesystem volumes
  creationTimestamp: "2023-12-19T11:26:35Z"
  name: ocs-external-storagecluster-cephfs
  resourceVersion: "3870482"
  uid: 383b3a6f-6404-472f-9a1c-2c3eb29ec274
parameters:
  clusterID: openshift-storage-extended
  [csi.storage.k8s.io/controller-expand-secret-name](http://csi.storage.k8s.io/controller-expand-secret-name): rook-csi-cephfs-provisioner
  [csi.storage.k8s.io/controller-expand-secret-namespace](http://csi.storage.k8s.io/controller-expand-secret-namespace): openshift-storage-extended
  [csi.storage.k8s.io/node-stage-secret-name](http://csi.storage.k8s.io/node-stage-secret-name): rook-csi-cephfs-node
  [csi.storage.k8s.io/node-stage-secret-namespace](http://csi.storage.k8s.io/node-stage-secret-namespace): openshift-storage-extended
  [csi.storage.k8s.io/provisioner-secret-name](http://csi.storage.k8s.io/provisioner-secret-name): rook-csi-cephfs-provisioner
  [csi.storage.k8s.io/provisioner-secret-namespace](http://csi.storage.k8s.io/provisioner-secret-namespace): openshift-storage-extended
  fsName: fsvol001
  pool: cephfs.fsvol001.data
provisioner: [openshift-storage.cephfs.csi.ceph.com](http://openshift-storage.cephfs.csi.ceph.com/)
reclaimPolicy: Delete
volumeBindingMode: Immediate